### DISCUSSION TITLE
Original youtube video link/url

### DISCUSSION THREAD

**Discussion by wagb283** [FreeTubeApp/FreeTube#4290] [Q&A]
How do I access the actual url/link of a video? I found one previous discussion on this in which the dev (PrestonN?) said it could easily be added to the side-bar, but as a new user, I couldn't find it.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @wagb283 see vid


https://github.com/FreeTubeApp/FreeTube/assets/73130443/2d050a7c-1706-4a29-ba46-1451960dc968

**Reply by wagb283** [reply]
Thanks! I don't know how I missed that...but the toggle's a lot cooler than the earlier proposed side-bar selection!
